packages/blocks/package.json

```diff
@@ -30,7 +30,11 @@
 		"typecheck": "tsc --noEmit"
 	},
 	"dependencies": {
-		"html-entities": "^2.5.2",
+		"html-entities": "^2.5.2"
+	},
+	"peerDependencies": {
+		"@snapwp/core": "^0.3.1",
```

Comment:
```suggestion
		"@snapwp/core": "*",
```

Reply:
Rationale for a specific number:

1. This is a peer dep, which means users don't have our root `package.json` to infer the values from.
2. This is a (probably invalid currently) _minimum_ version dependency. While I used the latest version for ease, this number will be blocked if/when `@blocks` requires a new minimum version of `@core`